sdk/python/.pylintrc

```diff
@@ -11,7 +11,7 @@ ignore=CVS
 
 # Add files or directories matching the regex patterns to the blacklist. The
 # regex matches against base names, not paths.
-ignore-patterns=.*_pb2.*
+ignore-patterns=.*_pb2.py,.*_pb2_grpc.py
```

Comment:
I thought this would tell pylint to look at the pyi files, but it doesn't seem to do anything with them yet.